ibm/mas_devops/roles/suite_app_install/templates/operator-group.yml.j2

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: {{ mas_app_namespace }}-operator-group
+  name: ibm-{{ mas_app_namespace }}-operatorgroup
```

Comment:
To avoid this being a breaking change in the collection, the code needs to be able to work with the old naming convention too.

e.g we know there can be only one operatorgroup in the namespace ... do a lookup for an operatorgroup, and if we find one, just use that name.  But if we don't use this name.  This way the collection will still work if you use it to update a previously installed version of MAS using the collection.  CoreAPI should behave the same way too as well, we shouldn't be so fragile to the name of the operatorgroup.

Reply:
@durera I have moved this change to the draft PR https://github.com/ibm-mas/ansible-devops/pull/317 so we can go ahead and merge this one sooner to bring back daily fvts as expected by João and team.